src/components/Notification.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */ 
+import { FC, useEffect } from 'react';
+import { Error } from '../utils/Error';
+
+type Props = {
+  errorMessage: Error ;
+  showNotification: boolean
+  setShowNotification: (showNotification: boolean)=> void;
+};
+
+export const Notification: FC <Props> = ({
+  errorMessage, showNotification, setShowNotification,
+}) => {
+  useEffect(() => {
+    if (showNotification) {
```

Comment:
you may store this value here, in the `Notification` component
and put `errorMessage` in the dependency array

Reply:
what do you mean?